src/librustdoc/html/render.rs

```diff
@@ -1893,8 +1914,9 @@ fn render_assoc_const_value(item: &clean::Item) -> String {
 
 fn document_full(w: &mut fmt::Formatter, item: &clean::Item,
                  cx: &Context, prefix: &str) -> fmt::Result {
-    if let Some(s) = item.doc_value() {
-        render_markdown(w, s, item.source.clone(), cx.render_type, prefix, &cx.shared)?;
+    if let Some(s) = cx.shared.maybe_collapsed_doc_value(item) {
+        trace!("Doc block: =====\n{}\n=====", s);
```

Comment:
Shouldn't `debug!` be used instead? (Don't really know the good macro we're supposed to use in such cases...)

Reply:
Probably? I'm not that solid on what counts as what either, but i'll go ahead and switch it over.